Explorer/Assets/DCL/Multiplayer/Movement/Settings/Rules/VelocitySpeedTiersRule.cs

```diff
@@ -0,0 +1,45 @@
+﻿using DCL.CharacterMotion.Components;
+using UnityEngine;
+
+namespace DCL.Multiplayer.Movement.Settings
+{
+    public class VelocitySpeedTiersRuleBase : SendRuleBase
```

Comment:
Is this class ever used?

Reply:
These classes where created to test at runtime different sending rules and fine-tune them. For current PR we stopped developing this part and use only one rule, which sends messages in a constant interval. We will come back to optimize and polish this part in next branches (after merge). Then we will drop some of them